<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyAudit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Pagella</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EasyAudit</string>
    </property>
   </widget>
   <widget class="QLabel" name="OS">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="OSVersion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Hostname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="KernalVersion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="MachineType">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Benchmark">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CIS Benchmark</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Server">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Workstation">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>450</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Workstation</string>
    </property>
   </widget>
   <widget class="QLabel" name="Processor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
